ow_plexil/src/plans/HealthMonitor.plp

```diff
@@ -44,61 +50,69 @@ HealthMonitor:
                   CameraOperable &&
                   PowerOperable;
 
-    if !ArmOperable && PrevArm{
-      if Lookup(IsFaulty("Arm")){
-        log_warning("Health Monitor: Arm is Inoperable due to the local fault(s): ", Lookup(ActiveFaults("Arm")));
+    if !ArmOperable && PrevArm {
+      if Lookup(IsFaulty("Arm")) {
+        log_warning("Health Monitor: Arm is Inoperable due to the local fault(s): ",
+                    Lookup(ActiveFaults("Arm")));
       }
-      else{
+      else {
         log_warning("Health Monitor: Arm is Inoperable due to non-local fault(s).");
       }
       PrevArm = false;
```

Comment:
I am unclear on what the `Prev*` variables represent. They could use a comment explaining them or a more descriptive name.

Reply:
I had the same feeling.  Though verbose, I have renamed `PrevArm` to `ArmPreviouslyWorking` to make the meaning clear.  I renamed the other flags accordingly and added a comment about them.